doc/samtools-mpileup.1

```diff
@@ -142,6 +142,10 @@ reference bases represented similarly.  (Subsequent pileup lines will
 contain \(lq*\(rq for this read indicating the deleted bases.)
 .IP \(bu 2
 If this is the last position covered by the read, a \(lq$\(rq character.
+.IP \(bu 2
```

Comment:
Each of these subbullet items is a character(s) that appears in the output. So this would be better beside the table it relates to (and without its own bullet) rather than lost down the bottom of the list of items.